src/java.desktop/share/classes/java/awt/image/ComponentColorModel.java

```diff
@@ -518,9 +518,6 @@ private void initScale() {
         case DataBuffer.TYPE_BYTE:
             {
                 byte[] bpixel = new byte[numComponents];
-                for (int i = 0; i < numColorComponents; i++) {
```

Comment:
Is it possible that indirectly this code threw the AIOBE for some wrong "numColorComponents"?